components/decimal/src/lib.rs

```diff
@@ -214,4 +214,38 @@ impl FixedDecimalFormatter {
     pub fn format_to_string(&self, value: &SignedFixedDecimal) -> String {
         self.format(value).write_to_string().into_owned()
     }
+
+    /// Gets the resolved numbering system identifier of this formatter.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use icu::decimal::FixedDecimalFormatter;
+    /// use icu::locale::locale;
+    ///
+    /// let fmt_en = FixedDecimalFormatter::try_new(
+    ///     locale!("en").into(),
+    ///     Default::default()
+    /// )
+    /// .unwrap();
+    ///
+    /// let fmt_bn = FixedDecimalFormatter::try_new(
+    ///     locale!("bn").into(),
+    ///     Default::default()
+    /// )
+    /// .unwrap();
+    ///
+    /// let fmt_zh_nu = FixedDecimalFormatter::try_new(
+    ///     locale!("zh-u-nu-hanidec").into(),
+    ///     Default::default()
+    /// )
+    /// .unwrap();
+    ///
+    /// assert_eq!(fmt_en.numbering_system(), "latn");
+    /// assert_eq!(fmt_bn.numbering_system(), "beng");
+    /// assert_eq!(fmt_zh_nu.numbering_system(), "hanidec");
+    /// ```
+    pub fn numbering_system(&self) -> String {
```

Comment:
Question: `String` or `&str` or `TinyAsciiStr` or something else? (you know all the pros and cons)

@Manishearth

Reply:
not String, either &str, Value, or TinyAsciiStr, preference on the latter two.

Reply:
Made it return `Value`.